bridge/slack/slack.go

```diff
@@ -483,7 +487,10 @@ func (b *Bslack) prepareMessageParameters(msg *config.Message) *slack.PostMessag
 			params.Attachments = append(params.Attachments, attach.([]slack.Attachment)...)
 		}
 	}
-	return &params
+	var opts []slack.MsgOption
+	opts = append(opts, slack.MsgOptionAttachments(params.Attachments...))
+	opts = append(opts, slack.MsgOptionPostMessageParameters(params))
```

Comment:
`params` (`PostMessageParameters`) shouldn't contain `Attachments` anymore (https://github.com/nlopes/slack/blob/master/chat.go#L45)

So maybe it's better to create a separate var `attachments` instead of `params.Attachments` to use in the `MsgOptionAttachments` function